dev/ansible/01_build_docker_image.yml

```diff
@@ -0,0 +1,18 @@
+- name: Remove docker image
```

Comment:
What happens if don't want to use the local docker image? Will it still work?

Reply:
I don't think it´s a problem.

Reply:
Have you tested it?